curriculum/challenges/english/25-front-end-development/workshop-greeting-card/67333082ab35e414eb4ea70d.md

```diff
@@ -0,0 +1,91 @@
+---
+id: 67333082ab35e414eb4ea70d
+title: Step 14
+challengeType: 0
+dashedName: step-14
+---
+
+# --description--
+
+Add to the `.card` selector `transition: transform 0.3s, background-color 0.3s ease`.
```

Comment:
Same here. Even though `transition` was briefly mentioned in the last lecture video, lets add a short reminder note here about how this code is working.